package.json

```diff
@@ -16,7 +16,8 @@
     "husky": "^4.3.0",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^26.5.2",
-    "lerna": "^3.22.1",
+    "lerna": "^4.0.0",
```

Comment:
Getting the following error after installing deps and running `npm run start in the `/ui` folder
![image](https://user-images.githubusercontent.com/13898415/143844481-401109f6-3cc1-4288-b935-4cffa1958a2e.png)

Reply:
fixed